src/main/java/com/zextras/persistence/converters/SlcwConverter.java

```diff
@@ -0,0 +1,56 @@
+package com.zextras.persistence.converters;
+
+import com.unboundid.ldap.sdk.Attribute;
+import com.unboundid.ldap.sdk.Modification;
+import com.unboundid.ldap.sdk.ModificationType;
+import com.zextras.persistence.mapping.entries.SlcwEntry;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Helper class that performs converting operations.
```

Comment:
Helper class! Please add a private constructor to prevent creation of object instances.

Reply:
done!